docs/merged_objects.md

```diff
@@ -46,6 +46,186 @@ rownames(merged_sce) # matrix row names
+| `submitter_celltype_annotation`  | If available, cell type annotations obtained from the group that submitted the original data. Cells that the submitter did not annotate are labeled as `"Submitter-excluded"`                                                                                                                                                                                   |
+| `singler_celltype_annotation`    | If cell typing with `SingleR` was performed, the annotated cell type. Cells labeled as `NA` are those which `SingleR` could not confidently annotate. If cell typing was performed for some but not all libraries in the merged object, libraries without annotations will be labeled `"Cell type annotation not performed"`                                    |
+| `singler_celltype_ontology`      | If cell typing with `SingleR` was performed with ontology labels, the annotated cell type's ontology ID. Cells labeled as `NA` are those which `SingleR` could not confidently annotate. If cell typing was performed for some but not all libraries in the merged object, libraries without annotations will be labeled `"Cell type annotation not performed"` |
+| `cellassign_celltype_annotation` | If cell typing with `CellAssign` was performed, the annotated cell type. Cells labeled as `"other"` are those which `CellAssign` could not confidently annotate. If cell typing was performed for some but not all libraries in the merged object, libraries without annotations will be labeled `"Cell type annotation not performed"`                         |
+| `cellassign_max_prediction`      | If cell typing with `CellAssign` was performed, the annotation's prediction score (probability)                                                                                                                                                                                                                                                                 |
+
+
+Unlike for {ref}`individual SCE objects<sce_file_contents:singlecellexperiment cell metrics`, cluster assignments are not included in the `colData`.
+
+
+### SingleCellExperiment gene information and metrics
+
+Gene information and metrics calculated from the RNA-seq expression data are stored as a `DataFrame` in the `rowData` slot, with the Ensembl ID as the names of the rows.
+
+```r
+rowData(merged_sce) # gene metrics
+```
+
+The following columns are included for all genes.
+The columns `mean` and `detected` will appear for each library id included in the merged object, named as shown in the table below.
+However, there will only be a single `gene_symbol` column, as this information equally pertains to all libraries.
```

Comment:
```suggestion
However, there will only be a single `gene_symbol` and `gene_id` column, as this information equally pertains to all libraries.
```
Is it `gene_id` or `gene_ids`, I think `gene_id`?

Reply:
Ah, `gene_ids`! I'll add this on my end.

```
> sce <- readRDS("SCPCL000001_processed.rds")
> names(rowData(sce))
[1] "gene_ids"    "gene_symbol" "mean"        "detected"  
```